task2/client_src/containers/game/game.js

```diff
@@ -0,0 +1,88 @@
+var GodGiftForm = require('containers/god-gift-form/god-gift-form.js');
+var UserWealthBlock = require('containers/user-wealth/user-wealth.js');
+var GiftTunnerBlock = require('containers/gift-tunner/gift-tunner.js');
+var GodIndicator = require('containers/god-indicator/god-indicator.js');
+var ResourceModel = require('models/resource.js');
+var IndicatorModel = require('models/indicator.js');
+
+module.exports = function Game() {
+
+	var elem = $('<div></div>');
+	var initialResourcesDataArray = [];
+	var godGiftForm, userWealthBlock;
+
+	// Getting initial resource data from local JSON server
+	var initPromise = fetch('/json-server/wealth/')
+		.then(function(response) {
+			return response.json();
+		})
+		.then(function(result) {
+			initialResourcesDataArray = result.resources;
```

Comment:
I'd suggest to not create `initialResourcesDataArray` variable, but pass `result.resources` into function, like this `gameBlockCreation(result.resources)`

It will be more declarative, meaning "function `gameBlockCreation` depends of one parameter `resources`"